app/src/main/java/com/example/pocketcloset/SaveClothingView.java

```diff
@@ -0,0 +1,176 @@
+        ivSavedPhoto = findViewById(R.id.ivSavedPhoto);
+        toggleButton = findViewById(R.id.toggleButton);
+        btnAccessories = findViewById(R.id.btnAccessories);
+        btnTops = findViewById(R.id.btnTop);
+        btnHeadwear = findViewById(R.id.btnHeadwear);
+        btnOverwear = findViewById(R.id.btnOverwear);
+        btnBottoms = findViewById(R.id.btnBottoms);
+        btnShoes = findViewById(R.id.btnShoes);
+        toggleButtonBottoms = findViewById(R.id.toggleButtonBottoms);
+        toggleButtonBottoms.setVisibility(View.GONE);
+
+        toggleButton.addOnButtonCheckedListener(new MaterialButtonToggleGroup.OnButtonCheckedListener() {
+            @Override
+            public void onButtonChecked(MaterialButtonToggleGroup group, int checkedId, boolean isChecked) {
+                if (checkedId == R.id.btnBottoms) {
+                    toggleButtonBottoms.setVisibility(View.VISIBLE);
+                    toggleButton.setVisibility(View.GONE);
+                }
+                ibAdd.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        Log.i("SaveClothingView", "Add Button Clicked");
+
+                        switch (checkedId) {
+                            case R.id.btnTop:
+                                addClothing("Top");
+
+                                break;
+
+                            case R.id.btnBottoms:
+                                addClothing("Pants");
+                                break;
+
+                            case R.id.btnShoes:
+                                addClothing("Shoes");
+
+                                break;
+
+                            case R.id.btnHeadwear:
+
+                                addClothing("Headwear");
+                                break;
+
+                            case R.id.btnOverwear:
+
+                                addClothing("Overwear");
+
+                                break;
+                            case R.id.btnAccessories:
+                                addClothing("Earrings");
+
+                        }
+
+                    }
+                });
+
+            }
+        });
+
+        toggleButtonBottoms.addOnButtonCheckedListener(new MaterialButtonToggleGroup.OnButtonCheckedListener() {
+            @Override
+            public void onButtonChecked(MaterialButtonToggleGroup group, int checkedId, boolean isChecked) {
+                if (checkedId == R.id.btnBottoms) {
+                    toggleButtonBottoms.setVisibility(View.VISIBLE);
+                    toggleButton.setVisibility(View.GONE);
+                }
+                ibAdd.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        switch (checkedId) {
+                            case R.id.btnSkirt:
+                                addClothing("Skirt");
```

Comment:
Consider creating a class with constants for the various types of clothing so that you don't have to use string literals throughout. This will help prevent typos.